custom_components/ramses_cc/broker.py

```diff
@@ -67,8 +68,6 @@ def __init__(self, hass: HomeAssistant, hass_config: ConfigType) -> None:
         self._rems = []
         self._zons = []
 
-        self.loop_task = None
-        self._last_update = dt.min
```

Comment:
This was set on every update but not otherwise used so removing accordingly.

Reply:
a remnant from test/dev - I usually mark these with `# TODO`